paper/paper.bib

```diff
@@ -0,0 +1,26 @@
+
+@online{frictionlessdata_datapackage,
+  author = {frictionlessdata},
+  title = {DataPackage-py: A Python library for working with Data Packages},
+  year = {2017},
+  url = {https://github.com/frictionlessdata/datapackage-py}
+}
```

Comment:
I'm not sure if we need a citation to their Python package.

Reply:
Right, I had that there since retriever was using the DataPackage model, but I can definitely take it off!